compiler/src/test/scala/org/scalajs/nscplugin/test/JSGlobalScopeTest.scala

```diff
@@ -398,6 +398,27 @@ class JSGlobalScopeTest extends DirectTest with TestHelpers {
     }
   }
 
+  @Test
+  def acceptGlobalThis(): Unit = {
```

Comment:
This does not seem to be necessary. It is already (better) covered by the tests in the `test-suite`.